woocommerce/woocommerce-gutenberg-products-block
# WIP: Run E2E test suite on WordPress 6.4

## What

This PR fixes the failing E2E tests when using WordPress 6.4

## Why



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Confirm all tests are passing in this PR


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add suggested changelog entry here.
